src/ui/public/saved_objects/saved_objects_client.js

```diff
@@ -69,37 +141,27 @@ export class SavedObjectsClient {
   }
 
   /**
-   * @param {string} type
-   * @param {object} attributes
-   * @returns {promise}
-  */
-  create(type, attributes) {
-    if (!type || !attributes) {
-      return this._PromiseCtor.reject(new Error('requires type and attributes'));
-    }
+   * Throttled processing of get requests into bulk requests at 100ms interval
+   */
+  processBatchQueue = throttle(() => {
+    const queue = cloneDeep(this.batchQueue);
+    this.batchQueue = [];
 
-    const url = this._getUrl([type]);
+    this.bulkGet(queue).then(({ savedObjects }) => {
+      queue.forEach((queueItem) => {
+        const foundObject = savedObjects.find(savedObject => {
+          return savedObject.id === queueItem.id & savedObject.type === queueItem.type;
+        });
 
-    return this._request('POST', url, { attributes });
-  }
+        if (!foundObject) {
+          return queueItem.resolve(this.createSavedObject(pick(queueItem, ['id', 'type'])));
+        }
 
-  /**
-   * @param {object} options
-   * @param {string} options.type
-   * @param {string} options.search
-   * @param {integer} options.page
-   * @param {integer} options.perPage
-   * @param {array} option.fields
-   * @returns {promise}
-   */
-  find(options = {}) {
-    const url = this._getUrl([options.type], keysToSnakeCaseShallow(options));
-
-    return this._request('GET', url).then(resp => {
-      resp.saved_objects = resp.saved_objects.map(d => this.createSavedObject(d));
-      return keysToCamelCaseShallow(resp);
+        queueItem.resolve(foundObject);
+      });
     });
-  }
+
+  }, BATCH_INTERVAL, { leading: false })
```

Comment:
should have a `;` at the end because it's an assignment (my intellij gives me a little complaining highlight)